internal/hcsoci/resources_wcow.go

```diff
@@ -145,7 +145,7 @@ func setupMounts(ctx context.Context, coi *createOptionsInternal, r *resources.R
 						readOnly,
 						coi.HostingSystem.ID(),
 						"",
-						&scsi.MountConfig{},
+						&scsi.MountConfig{ContainerMount: true},
```

Comment:
The `ContainerMount` field isn't applicable to WCOW GCS, right? Should we be setting it here?